Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6916,14 +6916,44 @@ def get_security_profiles_command(security_profile: str = None):
 
 
 @logger
-def apply_security_profile(xpath: str, profile_name: str) -> Dict:
+def apply_security_profile(xpath: str, profile_name: str, profile_type: str) -> Dict:
+    # first api call just for getting the current status
+    params = {
+        'action': 'get',
+        'type': 'config',
+        'xpath': xpath,
+        'key': API_KEY,
+    }
+    result = http_request(URL, 'GET', params=params)
+
+    # Get all profile types already existing, so we don't override them when updating
+    profile_types_result = dict_safe_get(result, ['response', 'result', 'entry', 'profile-setting', 'profiles'],
+                                         default_return_value={})
+
+    # keep the context aligned with both committed and un-committed objects
```

Comment:
```suggestion
    # align the response for both committed and un-committed profiles 
```

this isn't needed for the context as we dont return context in this command.